packages/console/src/containers/AppContent/components/UserInfo/index.tsx

```diff
@@ -36,16 +36,14 @@ const UserInfo = () => {
 
   const [isLoading, setIsLoading] = useState(false);
   const {
-    data: { language, appearanceMode },
+    data: { language = 'en', appearanceMode = DynamicAppearanceMode.System },
```

Comment:
The `appearanceMode` has a default value in the user preferences, so we should remove `DynamicAppearanceMode.System` here.